src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -228,6 +230,61 @@ private void emitDirectoryChanged(UUID directoryUuid, String elementName, String
             .then();
     }
 
+    public Mono<Void> updateElementDirectory(UUID elementUuid, UUID newDirectoryUuid, String userId) {
+        ElementAttributes oldParent = getParentElement(elementUuid);
```

Comment:
What happens if `elementUuid` is a root directory one ?
getParentElement will return null ? then it will fail when `oldParent` is used

Reply:
fixed + new test to cover the case